src/Components/Shared/NavBar/NavBar.tsx

```diff
@@ -1,7 +1,41 @@
-import React from 'react'
+import AppBar from "@mui/material/AppBar";
 
-export default function NavBar() {
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+
+import { Avatar, Link, Stack } from "@mui/material";
+
+export default function NavBar({ adminData }) {
+  //  console.log(adminData);
+  const drawerWidth = 240;
   return (
-    <div>NavBar</div>
-  )
-}
+    <>
+      <AppBar
+        sx={{ width: `calc(100% - ${drawerWidth}px)`, ml: `${drawerWidth}px` }}
+        position="static"
+      >
+        <Toolbar>
+          <Link
+            sx={{ flexGrow: 1 }}
+            href="/dashboard"
+            color="inherit"
+            underline="hover"
+          >
+            {" "}
+            Booking
+          </Link>
+
+          <Typography variant="" color="initial" sx={{ mr: 4 }}>
+            {adminData.userName}
```

Comment:
We need to add the optional chaining operator to this to avoid the app breaking when the `adminData` is initially null `{adminData?.userName}`